<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>396</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>396</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Library</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>256</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Type document:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_getType">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>All types</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Book</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Abstract</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Newspaper</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Magazine</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Map</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Manuscript</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Name document:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="l_getName">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Author:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="l_getAuthor">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Date of creation:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="de_getDate">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Address:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="l_getAddress">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="b_search">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b_changeDocuments">
          <property name="text">
           <string>Add or delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Type document:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_setType">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Book</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Abstract</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Newspaper</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Magazine</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Manuscript</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Map</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Name document:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="l_setName">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Author:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="l_setAuthor">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Date of creation:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="de_setDate">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Address:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="l_setAddress">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="b_addDocuments">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b_deleteDocuments">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b_return">
          <property name="text">
           <string>Go to search</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
